### ISSUE TITLE
Quill Editor fails accessibility tests

### ISSUE THREAD

**Issue by brentswisher** [brentswisher/anywhere-editor#1]
There are some minor issues with the quill editor that prevent it from passing the automated axe accessibility tests, we should either find a way to fix them or migrate to another rich text editor that does pass accessibility.
